lib/fathead/django_built_ins/parse.py

```diff
@@ -0,0 +1,177 @@
+from bs4 import BeautifulSoup
+
+DJANGO_HOME = 'https://www.djangoproject.com/'
+DJANGO_DOC_URL = 'https://docs.djangoproject.com/en/1.9/ref/templates/builtins/'
```

Comment:
@milmazz I made this change, but I think the more important change here would be to pass an argument in the `fetch.sh` script since that is what downloads the docs for the version. That and this will need to be in sync.